src/amo/pages/UserProfileEdit/index.js

```diff
@@ -722,6 +722,7 @@ export class UserProfileEditBase extends React.Component<Props, State> {
               </p>
 
               <UserProfileEditNotifications
+                key={user && user.id}
```

Comment:
This is an edge case, I noticed this component does not update the UI when switching from one user to another (which is super tricky to reproduce), even if all the props/data are correct.